edit/edit.go

```diff
@@ -184,20 +184,23 @@ func ExistingPackageDeclaration(f *build.File) *build.Rule {
 
 // PackageDeclaration returns the package declaration. If it doesn't
 // exist, it is created at the top of the BUILD file, after leading
-// comments.
+// comments / load statements.
 func PackageDeclaration(f *build.File) *build.Rule {
 	if pkg := ExistingPackageDeclaration(f); pkg != nil {
 		return pkg
 	}
 	all := []build.Expr{}
 	added := false
 	call := &build.CallExpr{X: &build.Ident{Name: "package"}}
-	// Skip CommentBlocks and find a place to insert the package declaration.
 	for _, stmt := range f.Stmt {
-		_, ok := stmt.(*build.CommentBlock)
-		if !ok && !added {
-			all = append(all, call)
-			added = true
+		switch stmt.(type) {
+		case *build.CommentBlock, *build.LoadStmt:
```

Comment:
Perhaps it should also be added after a docstring if it exists? Not sure what's the formal definition (e.g. whether it must be the first statement in the file or can follow several comment blocks).

Reply:
Done (a string literal at top-level is a good approximation)